build.gradle.kts

```diff
@@ -1,52 +1,40 @@
+import org.jetbrains.changelog.Changelog
 import org.jetbrains.changelog.markdownToHTML
-import org.jetbrains.intellij.tasks.RunPluginVerifierTask.FailureLevel
-
-fun properties(key: String) = project.findProperty(key).toString()
-
-// Extract the <!-- Plugin description --> section from README.md and provide for the plugin's manifest
-val pluginDescriptionFromMd = File(projectDir, "README.md").readText().lines().run {
-    val start = "<!-- Plugin description -->"
-    val end = "<!-- Plugin description end -->"
-
-    if (!containsAll(listOf(start, end))) {
-        throw GradleException("Plugin description section not found in README.md:\n$start ... $end")
-    }
-    subList(indexOf(start) + 1, indexOf(end))
-}.joinToString("\n").run { markdownToHTML(this) }
-
+import org.jetbrains.intellij.platform.gradle.TestFrameworkType
 
 plugins {
-    // Java support
-    id("java")
-    // Kotlin support
-    id("org.jetbrains.kotlin.jvm") version "1.9.0"
-    // Gradle IntelliJ Plugin
-    id("org.jetbrains.intellij") version "1.13.3"
-    // Gradle Changelog Plugin
-    id("org.jetbrains.changelog") version "1.3.1"
-
-    kotlin("plugin.serialization") version "1.7.20"
+    id("java") // Java support
+    alias(libs.plugins.kotlin) // Kotlin support
+    alias(libs.plugins.intelliJPlatform) // IntelliJ Platform Gradle Plugin
```

Comment:
Which version might be used with this `alias` ? This could have caused the verify failure. Need to find alternative for the `listProductReleases` task in this new version.

Reply:
version information exists on libs.versions.toml

Reply:
> Need to find alternative for the `listProductReleases` task in this new version.

You might want to take a look at the updated version of the github action yml of the [plugin template](https://github.com/JetBrains/intellij-platform-plugin-template/blob/main/.github/workflows/build.yml)